src/AppInstallerSharedLib/Versions.cpp

```diff
@@ -437,12 +438,12 @@ namespace AppInstaller::Utility
 
     UInt64Version::UInt64Version(std::string&& version, std::string_view splitChars)
     {
-        Assign(std::move(version), splitChars);
+        Assign(std::move(Utility::Trim(version)), splitChars);

```

Comment:
Let `Assign` trim.  May require updating the custom `Assign` for this type.

Reply:
Thanks for the guidance! After reviewing the custom `Assign`, the method that takes in a string goes back to `Version::Assign`, so I'll let that handle the trimming, and the other method only takes in integers and shouldn't require trimming.